app/utils/theme.ts

```diff
@@ -5,7 +5,7 @@ export const theme = createTheme({
   palette: {
     mode: "dark",
     background: {
-      default: "#222222",
+      default: "#000029",
       paper: "#242526",
```

Comment:
Might want to check if this color needs to be changed too. Paper affects the card components on the events page. A good color matching the default but a little lighter would be nice